app/src/main/java/ru/thstdio/aa2020/cache/workmanager/CinemaWorkManager.kt

```diff
@@ -0,0 +1,33 @@
+package ru.thstdio.aa2020.cache.workmanager
+
+import android.content.Context
+import android.util.Log
+import androidx.work.Constraints
+import androidx.work.NetworkType
+import androidx.work.PeriodicWorkRequest
+import androidx.work.WorkManager
+import java.util.concurrent.TimeUnit
+
+
+object CinemaWorkManager {
+    private const val WORK_TAG = "CinemaWorkManager_Tag"
+    private val constraints = Constraints.Builder()
```

Comment:
move constraints, work to local variables of start fucntion
to not keep reference all time on unused object